templates/vue/src/components/Lightbox/media/ActivityMedia/TapestryActivity.vue

```diff
@@ -2,12 +2,13 @@
   <div class="tapestry-activity">
     <div class="icon"><tapestry-icon :icon="type" /></div>
     <div v-if="type === 'text'" class="text">{{ answerData }}</div>
-    <audio v-if="type === 'audio'" controls :src="answerData"></audio>
+    <audio v-if="type === 'audio'" controls :src="audioUrl"></audio>
```

Comment:
This should be answerData as it is. I see that you cannot get the audio now if you show previous answer to an audio activity.